unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs

```diff
@@ -469,21 +469,21 @@ public void SetEntityParent(string entityId, string parentId)
 
             if (me != null)
             {
-                if (parentId == "PlayerEntityReference")
+                if (parentId == "FirstPersonCameraEntityReference")
                 {
-                    me.SetParent(DCLCharacterController.i.playerReference);
+                    me.SetParent(DCLCharacterController.i.firstPersonCameraReference);
                     SceneController.i.boundariesChecker.AddPersistent(me);
                     SceneController.i.physicsSyncController.MarkDirty();
                 }
-                else if (parentId == "AvatarPositionEntityReference")
+                else if (parentId == "AvatarEntityReference")
                 {
-                    me.SetParent(DCLCharacterController.i.avatarPositionReference);
+                    me.SetParent(DCLCharacterController.i.avatarReference);
                     SceneController.i.boundariesChecker.AddPersistent(me);
                     SceneController.i.physicsSyncController.MarkDirty();
                 }
```

Comment:
Can we add here a couple of little comments about what is the supposed behavior when setting as parent any of those entities? So anybody can understand without needing to navigate through the code